src/controllers/expense.controller.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+const { expenseService } = require('../services/expense.service.js');
+const { userService } = require('../services/user.service.js');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  res.send(expenseService.getAll({
+    userId,
+    categories,
+    from,
+    to,
+  }));
```

Comment:
save result of the service function to some variable and then send variable, it will increase readability of your code